TombEngine/Game/control/lot.cpp

```diff
@@ -146,6 +146,12 @@ void InitialiseSlot(short itemNumber, bool makeTarget)
 
 			break;
 
+		case LotType::SnowmobileGun:
```

Comment:
What this new LOT type has to do with enemy firing system refactor? If you have introduced snowmobile gun enemy, it should be on a separate PR.

Reply:
It's to fix snowmobile driver, actually it could only go forward and drop/climb 1 click, in TR2, it was able to fall from more than 4 click but not climb 2 click.
it's a small think so i through to let it on this PR. unlike cowboy which was a new entity.